src/psycopmlutils/loaders/raw/load_coercion.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional
+
+import pandas as pd
+
+from psycopmlutils.loaders.raw.sql_load import sql_load
+from psycopmlutils.utils import data_loaders
+
+
+class LoadCoercion:
+    @data_loaders.register("coercion")
+    def coercion(
+        coercion_type: Optional[str] = None,
+        reason_for_coercion: Optional[str] = None,
+        n: Optional[int] = None,
+    ) -> pd.DataFrame:
+        """Load coarcion data.
+
+        Args:
+            coercion_type (str): Type of coercion, e.g. 'tvangsindlæggelse', 'bæltefiksering'. # noqa: DAR102
+            reason_for_coercion (str): Reason for coercion, e.g. 'farlighed'. # noqa: DAR102
+            n: Number of rows to return. Defaults to None.
+
+        Returns:
+            pd.DataFrame
+        """
+        view = "[FOR_tvang_alt_hele_kohorten_inkl_2021]"
+
+        sql = f"SELECT dw_ek_borger, datotid_start_sei, varighed FROM [fct].{view}"
+
+        if coercion_type and reason_for_coercion is None:
+
+            sql += f"WHERE typetekst_sei = '{coercion_type}'"
+
+        if coercion_type is None and reason_for_coercion:
+
+            sql += f"WHERE begrundtekst_sei = '{reason_for_coercion}'"
+
+        if coercion_type and reason_for_coercion:
+
+            sql += f"WHERE typetekst_sei = '{coercion_type}' AND begrundtekst_sei = '{reason_for_coercion}'"
+
+        df = sql_load(sql, database="USR_PS_FORSK", chunksize=None, n=n)
+
+        df.rename(
+            columns={"datotid_start_sei": "timestamp", "varighed": "value"},
+            inplace=True,
+        )
+
+        # msg.good(f"Loaded {print_str}")
+        return df.reset_index(drop=True)
+
+    def _aggregate_coercion(
+        coercion_types_lists: list,
```

Comment:
```suggestion
        coercion_types_list: list,
```

Rename with F2 in VSCode.